src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -40,7 +39,7 @@
     <ProjectJson>$(MsBuildThisFileDirectory)windows\project.json</ProjectJson>
     <ProjectLockJson>$(MsBuildThisFileDirectory)windows\project.lock.json</ProjectLockJson>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(IsNETNativeLibrary)' != 'true' AND '$(TargetsWindows)' == 'true'">
+  <PropertyGroup Condition="'$(TargetGroup)' != 'netcore50' AND '$(TargetsWindows)' == 'true'">
     <DefineConstants>>$(DefineConstants);SUPPORTS_WINDOWSIDENTITY</DefineConstants>
```

Comment:
There's a small problem here that @iamjasonp identified. There's an extra > after the <DefineConstants> which will prevent the NegotiateStream functionality from working.